src/map/pc.cpp

```diff
@@ -1890,6 +1904,73 @@ uint8 pc_isequip(map_session_data *sd,int n)
 	return ITEM_EQUIP_ACK_OK;
 }
 
+bool pc_lastpoint_instance( map_session_data& sd ){
```

Comment:
might as well use const ref, we don't modify sd here.

Reply:
Or even better, make this a const method

Reply:
We modify sd at the bottom of the function.